chain/epoch_manager/src/lib.rs

```diff
@@ -174,7 +176,10 @@ impl EpochManager {
 
         for proposal in last_block_info.all_proposals.iter().rev() {
             if !slashed_validators.contains_key(&proposal.account_id) {
-                if proposal.stake == 0 && !proposals.contains_key(&proposal.account_id) {
+                if proposal.stake == 0
+                    && !proposals.contains_key(&proposal.account_id)
+                    && *next_epoch_info.stake_change.get(&proposal.account_id).unwrap_or(&0) != 0
```

Comment:
what happens when a fisherman unstakes?
seems like fishermen don't get recorded in stake_change